ygen/gogen_test.go

```diff
@@ -1242,6 +1242,21 @@ func (t *Tstruct) GetOrCreateListWithKey(KeyLeafOne string, KeyLeafTwo int8) (*L
 	return v
 }
 
+// GetListWithKey retrieves the value with the specified key from
+// the ListWithKey map field of Tstruct.
+func (t *Tstruct) GetListWithKey(KeyLeafOne string, KeyLeafTwo int8) (*ListWithKey){
+
+  key := Tstruct_ListWithKey_Key{
+		KeyLeafOne: KeyLeafOne,
+		KeyLeafTwo: KeyLeafTwo,
+	}
+
+  if lm, ok := t.ListWithKey[key]; ok {
```

Comment:
Should the GetList getters also be nil receiver resilient? So that we can chain through empty lists.

Reply:
Good catch, thank you. Done.